packages/core/useKeyModifier/index.ts

```diff
@@ -37,7 +37,8 @@ export function useKeyModifier<Initial extends boolean | null>(modifier: KeyModi
   if (document) {
     events.forEach((listenerEvent) => {
       useEventListener(document, listenerEvent, (evt: KeyboardEvent) => {
```

Comment:
KeyboardEvent | MouseEvent

Reply:
how about this? `Event & { getModifierState: (modifier: KeyModifier) => boolean }`

Reply:
It's a internal type, that won't affect users so I don't think it need to be that general

Reply:
ok